src/components/Dropdown/Dropdown.tsx

```diff
@@ -214,26 +218,33 @@ type DropdownItemProps = HTMLProps<HTMLLIElement> & {
 /**
  * A dropdown item component.
  */
-const DropdownItem = memo(
-  forwardRef<HTMLLIElement, DropdownItemProps>(function MemoDropdownItem(
-    // The props typing has some difficulties with memo and forwardRef hoc
-    // eslint-disable-next-line react/prop-types
-    { children, isSelected, ...props }: DropdownItemProps,
-    ref,
-  ) {
-    return (
-      <li
-        tabIndex={0}
-        role="option"
-        ref={ref}
-        aria-selected={isSelected}
-        {...props}
-      >
-        {children} {isSelected && <Check width="20" height="20" />}
-      </li>
-    );
-  }),
-);
+const DropdownItem = memo(function DropdownItem({
+  children,
+  isSelected,
+  isDisplayed,
+  ...props
+}: DropdownItemProps) {
+  const ref = useRef<HTMLLIElement>(null);
+
+  // Focus the item if the dropdown is open and the item is already selected
+  useEffect(() => {
+    if (isSelected && isDisplayed) {
+      ref?.current?.focus();
```

Comment:
```suggestion
      ref.current?.focus();
```

Isn't ref never falsy?

Reply:
Yep you're right. It's a remnant of the previous forward ref hoc :)